github/orgs.go

```diff
@@ -70,9 +70,10 @@ type Organization struct {
 	ReposURL         *string `json:"repos_url,omitempty"`
 }
 
+// OrganizationInstallations represents Github app installations of an organization
```

Comment:
Please fix: "Github" => "GitHub" and add a period to the end of the sentence for the auto-generated Godocs.

Reply:
Done.